### ISSUE TITLE
JSON notifications are sent every 5 seconds

### ISSUE THREAD

**Issue by guibom** [philippnormann/chia-monitor#14]
When I use the JSON notification (`json://{hostname}`), the notifications get sent every 5 seconds or so. 
It doesn't seem to respect the `status_interval_minutes` value.

As a test, I've tried using the Telegram notification -- And that works as expected. So perhaps it's related to specific types of notifications.

Ideally there should also be a way to disable notifications entirely. I'm personally only interested in the prometheus/grapfana part, and used JSON notifications as way to have a valid configuration file. That's when I've noticed my web server was receiving these constant POST requests.

**Comment by philippnormann** [comment]
Hey @guibom, I tried to reproduce your error with the JSON notifications and suspect that the reason for you seeing those frequent notifications is that your web server didn't respond with a 200 status code, causing the notifier to retry every other second. With a server responding 200's to every POST, the monitor does respect the configured interval and works as expected.

The suggested config flag to disable notifications entirely is indeed very useful and was added in https://github.com/philippnormann/chia-monitor/commit/14181871493ae1ce724732fdeafb64b4ccc5ee38.
